src/tests/CoreMangLib/system/reflection/emit/DynMethodJumpStubTests/DynMethodJumpStubTests.csproj

```diff
@@ -5,6 +5,8 @@
     <CLRTestPriority>1</CLRTestPriority>
     <!-- Test unsupported outside of windows -->
     <CLRTestTargetUnsupported Condition="'$(TargetsWindows)' != 'true'">true</CLRTestTargetUnsupported>
+    <!-- Test uses Reflection.Emit -->
```

Comment:
It would be better to use ConditionalFact to encode dependencies on Ref.Emit and similar.

(I know we are inconsistent in runtime tests today.)

Reply:
> (I know we are inconsistent in runtime tests today.)

We don't use it for ref.emit at all. One advantage of doing it in the csproj is that we don't compile the test at all vs we compile it and there is just a runtime check that is false.